### PR TITLE
fix handling of letters a and z

### PR DESCRIPTION
**Pull request by jannic**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -113,6 +113,17 @@ mod tests {
 
         let decrypted = decrypt_any(&encrypted, amount);
 
+        assert_eq!(plain, decrypted);
+    }
+    #[test]
+    fn test_rot13_abc() {
```
Comment (jD91mZM2) [inline src/lib.rs:119]: But... We already have a rot13 test?

**Review by jD91mZM2 — CHANGES_REQUESTED** [review]
Whoops! Major oversight on my part. Thanks for fixing it!

File: src/lib.rs (hunk 1 above)
Comment (jD91mZM2) [reply src/lib.rs:119]: Actually... Meh whatever it can stay

Comment (jannic) [reply src/lib.rs:119]: Well sure, it wasn't meant as an example, it's the missing test case that would have caught the bug!
So, finally, the crypto library should have (near) complete test coverage :-)

**Pull request merged**